Dining/models.py

```diff
@@ -58,9 +62,10 @@ class DiningList(models.Model):
     purchaser = models.ForeignKey(User, related_name="dininglist_purchaser", blank=True, null=True,
                                   on_delete=models.SET_NULL)
 
-    kitchen_cost = models.DecimalField(decimal_places=2, verbose_name="kitchen cost per person", max_digits=4,
-                                       default=0.50, validators=[MinValueValidator(Decimal('0.00'))])
-    dinner_cost_total = models.DecimalField(decimal_places=2, verbose_name="total dinner costs", max_digits=5,
+    # Kitchen cost may not be changed after creation
```

Comment:
That's because in this transaction implementation, the dining costs are directly drawn from an account when someone joins a dining list. That means that when the kitchen cost would change later, new transactions must be created for everyone on the dining list to apply the new kitchen cost. This has issues: we need more code (always bad) and also things can arise like a user's balance can become too negative. So I would not want for these costs to be changeable in this implementation. There is also no user requirement for this and I don't see why we would need that (changing kitchen cost would actually also mean to notify users, otherwise they can get unpleasant surprises).

Note also that changed kitchen costs can always be applied to new dining lists.